src/legacy/dev-application/languages/es.php

```diff
@@ -16,86 +16,87 @@
-"Modules" => "Moduli",
-"Images" => "Slike",
-"Settings" => "Podešavanja",
-"Object Editors" => "Editori objekta",
-"View Page" => "Pogledaj stranu",
-"Style for this object:" => "Stajl ovog objekta",
-"Update object" => "Updejtuj objekat",
-"Page"  => "Strana",
-"Template" => "Templejt",
-"Empty Cache" => "Izprazni keš",
-"Backup Site" => "Bekap sajta",
-"Back" => "Prethodna",
-"Next" => "Sledeća",
-
+"Prva" => "Primera categoría",
+"First Name" => "Nombre",
+"Last Name" => "Apellido",
+"First name" => "Nombre",
+"Last name" => "Apellido",
+"Company" => "Compañía",
+"eMail address" => "Dirección de correo electrónico",
+"Email address" => "Dirección de correo electrónico",
+"Phone number" => "Número de teléfono",
+"Message" => "Mensaje",
+"Submit" => "Enviar",
+"Lokacija" => "Ubicación",
+
+"Message sent :<br />" => "Se ha enviado el siguiente mensaje :<br />",
+"<br />Mail has been sent! Thank you." => "<br />¡El correo ha sido enviado! Gracias.",
+"Value is required and can't be empty" => "El valor es obligatorio y no puede estar vacío",
+
+"Cenovnik" => "Lista de precios",
+"Services" => "Servicios",
+"Prices" => "Precios",
+"Search" => "Buscar",
+
+"Welcome" => "Bienvenido",
+"Welcome Guest" => "Bienvenido Invitado",
+
+"You need to Login!" => "¡Necesitas iniciar sesión!",
+"Account Information" => "Información de la cuenta",
+"Username" => "Nombre de usuario",
+"Password" => "Contraseña",
+"Repeat Password" => "Repetir contraseña",
+"Full Name" => "Nombre completo",
+"Update" => "Actualizar",
+"Register" => "Registrarse",
+"ChangePassword" => "Cambiar contraseña",
+"Change"  => "Cambiar",
+"New Password" => "Nueva contraseña",
+"New Password retype" => "Reescribir nueva contraseña",
+"The username you selected already exists." => "El nombre de usuario que seleccionaste ya existe.",
+"The email you entered already exists." => "El correo electrónico que ingresaste ya existe.",
+"The supplied passwords don't match." => "¡Las contraseñas no coinciden!",
+"Your account has been created, you have been provided with activation link, check your mail!" =>  "Tu cuenta ha sido creada, se te ha proporcionado un enlace de activación, ¡revisa tu correo!",
+"Your Account has been activated!You can now login!" => "!Tu cuenta ha sido activada! ¡Ahora puedes iniciar sesión!",
+"Your activation link is invalid!" => "!Tu enlace de activación es inválido!",
+
+"There is no user with these credentials!" => "!No hay usuario con estas credenciales!",
+
+"Your Account, " =>  "!Tu Cuenta, ",
+
+"You are not logged in!Please Login to see your account details!" => "!No has iniciado sesión! ¡Inicia sesión para ver los detalles de tu cuenta!",
+
+"Password reminder is sent to the provided email!" => "!Se ha enviado un recordatorio de contraseña al correo electrónico proporcionado!",
+
+// Captcha messages
+"Please verify you're a human" => "!Por favor verifica que eres humano!",
+
+// Captcha error
+'Captcha value is wrong' => '!El valor del captcha es incorrecto!',
```

Comment:
@gabrielmoris  could you please use double quotes for all the strings from line 79 to 101, it is inconsistent to have most of the strings in an array with " and then this couple of key-value pairs with '

Reply:
Ich habe es schon Getan!